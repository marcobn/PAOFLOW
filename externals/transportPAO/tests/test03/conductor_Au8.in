
  &INPUT_CONDUCTOR
    postfix='_Au8_PROJ'
    work_dir = './SCRATCH'
    dimC = 72
    calculation_type = 'bulk'
    ne = 800  
    emin = -12.0
    emax =   4.0 
    datafile_C = "./SCRATCH/gold8.save/atomic_proj.xml"
    transport_dir = 3
    delta = 0.0001
    do_orthoovp    = .TRUE.
  /

<HAMILTONIAN_DATA>

   <H00_C  cols="1-72" rows="1-72" />
   <H_CR   cols="1-72" rows="1-72" />

</HAMILTONIAN_DATA>

