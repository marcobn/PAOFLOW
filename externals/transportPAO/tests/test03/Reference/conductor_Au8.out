
  ======================================================================
              =                                            =
              =     *** WanT *** Wannier Transport Code    =
              =        (www.wannier-transport.org)         =
              =      Ultra Soft Pseudopotential Implem.    =
              =                                            =
  ======================================================================


  Program <conductor>  v. 2.5.0alpha2  starts ...
  Date 17Apr2014 at 20:32:50 

     Number of MPI processes:       6



          BUILT :    Thu 17 Apr 2014 06:13:20 PM CEST
           HOST :    x86_64-unknown-linux-gnu
           ARCH :    x86_64
             CC :    icc
            CPP :    cpp
            F90 :    mpif90
            F77 :    ifort
         DFLAGS :    -D__INTEL -D__FFTW -D__MPI -D__PARA
      BLAS LIBS :    -L/home/ferretti/want/install -lmkl_em64t_nothread
    LAPACK LIBS :    
       FFT LIBS :    
      MASS LIBS :    



  ======================================================================
  =  Conductor Init                                                    =
  ======================================================================

  ./SCRATCH/gold8.save/atomic_proj.xml file fmt: atmproj
   ATMPROJ conversion to be done using: 
     atmproj_nbnd :     80
     atmproj_thr  :      0.900000
     atmproj_sh   :     10.000000

  ./SCRATCH/gold8.save/atomic_proj.xml converted from ATMPROJ to internal fmt

  ======================================================================
  =  INPUT Summary                                                     =
  ======================================================================

  <INPUT>
          Calculation Type :     bulk
                    prefix :     
                   postfix :     _Au8_PROJ
                  work_dir :     ./SCRATCH
               L-lead dim. :        72
            conductor dim. :        72
               R-lead dim. :        72
       Conductance Formula :     landauer
                  Carriers :     electrons
       Transport Direction :         3
          Have Correlation :     FALSE
              Write k-data :     FALSE
            Write sgm lead :     FALSE
                Write gf C :     FALSE
           Max iter number :       200
                    nprint :        20

        Conductor datafile :     ./SCRATCH/gold8.save/atomic_proj.xml.ham
       leads are identical :     TRUE 
         ovp orthogonaliz. :     TRUE 
  </INPUT>


  <ENERGY_GRID>
                 Dimension :        800
                 Buffering :          1
                Min Energy :      -12.00000
                Max Energy :        4.00000
               Energy Step :        0.02003
                     Delta :        0.00010
             Smearing Type :     lorentzian
             Smearing grid :      20000
             Smearing gmax :       25.00000
                   Shift_L :        0.00000
                   Shift_C :        0.00000
                   Shift_R :        0.00000
                Shift_corr :        0.00000
  </ENERGY_GRID>


  <K-POINTS>
       nkpts_par =    1
       nrtot_par =    1
        use_symm = TRUE

       Parallel kpoints grid:        nk = (  1  1  1 )   s = (  0  0  0 )
       k (   1) =    (   0.00000  0.00000  0.00000 ),   weight =   1.0000

       Parallel R vector grid:       nr = (  1  1  1 )
       R (   1) =    (   0.00000  0.00000  0.00000 ),   weight =   1.0000
  </K-POINTS>


  <PARALLELISM>
       Paralellization over frequencies
       # of processes:     6
  </PARALLELISM>


 <MEMORY_USAGE>
                smearing:          0.480 MB
                 kpoints:          0.000 MB
        hamiltonian data:          1.247 MB
        correlation data:          0.003 MB
               workspace:          0.000 MB

     Total alloc. Memory:          1.730 MB
 </MEMORY_USAGE>


  ======================================================================
  =  Frequency Loop                                                    =
  ======================================================================

  Computing E(     1 ) =    -12.00000 eV
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :     1.25 secs

  Computing E(    20 ) =    -11.61952 eV
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :     1.60 secs

  Computing E(    40 ) =    -11.21902 eV
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :     1.96 secs

  Computing E(    60 ) =    -10.81852 eV
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :     2.31 secs

  Computing E(    80 ) =    -10.41802 eV
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :     2.67 secs

  Computing E(   100 ) =    -10.01752 eV
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :     3.03 secs

  Computing E(   120 ) =     -9.61702 eV
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :     3.39 secs

  Computing E(   134 ) =     -9.33667 eV
  T matrix converged after avg. # of iterations      0.500

                              Total time spent up to now :     3.64 secs


  ======================================================================
  =  Writing data                                                      =
  ======================================================================


  conductance written on file:    cond_Au8_PROJ.dat

  doscond written on file:    doscond_Au8_PROJ.dat

  ======================================================================

   <global routines>
             clock number :    15

                conductor :    12.00s CPU 

           datafiles_init :     1.18s CPU
      atmproj_to_internal :     0.88s CPU
             compute_rham :     0.00s CPU (       5 calls,   0.000 s avg)
            smearing_init :     0.02s CPU
                   cft_1z :     0.01s CPU (       2 calls,   0.005 s avg)
         hamiltonian_init :     0.01s CPU
              read_matrix :     0.01s CPU (       2 calls,   0.003 s avg)
              fourier_par :     0.00s CPU (       4 calls,   0.000 s avg)
             do_conductor :    11.55s CPU
        hamiltonian_setup :     0.02s CPU (     134 calls,   0.000 s avg)
            transfer_mtrx :     1.56s CPU (     134 calls,   0.012 s avg)
                    green :     0.31s CPU (     268 calls,   0.001 s avg)
              gzero_maker :     0.03s CPU (     402 calls,   0.000 s avg)
            transmittance :     0.17s CPU (     134 calls,   0.001 s avg)


